spec/models/picture_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+RSpec.describe Picture, type: :model do
+  let(:user) { User.create email: 'testuser@email.com', password: "password", password_confirmation: "password" }
+
+  it { should belong_to :user }
+
+  it { should validate_presence_of :user }
+
+  it { should validate_presence_of :title }
+
+  it { should have_attached_file :image }
+
+  it { should validate_attachment_presence :image }
+
+  it { should validate_attachment_size(:image).
+                less_than(1.megabyte) }
```

Comment:
Expression at 17, 39 should be on its own line.<br>Use 2 (not 14) spaces for indenting an expression spanning multiple lines.